app/services/csv_import/companies.rb

```diff
@@ -37,6 +37,7 @@ def prepare_company(row)
     #
     def company_attributes(row)
       {
+        id: row[:id],
```

Comment:
I think we might include this mapping `id: row[:id]` in all csv importers.

Please consider the following scenario:

1. User exports CSV (`id` column are present in the file)
2. The user deletes one of the records in the application (it is actually soft-deleted)
3. The user then re-imports CSV with `id` column - he would expect that those ids from the file are going to be used and effectively he'd update deleted record, not create a new one.

Without the mapping, we'd re-create the same record but with different id - I guess we don't want that, we want that CSV import would preserve discarded records just as they are, right?

Reply:
@simaob @tsubik what do you think?

Reply:
removed this for now, let's move the discussion to another story.